### ISSUE TITLE
Update the pseudolabel

### ISSUE THREAD

**Issue by weiziquan** [lich0031/AIDE#3]
In the NNL task, the quality of the pseudolabel is actually quite high, so whether it is necessary to add a judgment before updating the pseudolabel. If the difference between the generatedmask output by net and the pseudolabel in the current epoch is too large, that is, the traincasediceavgtemp is very low, then the generatedmask output by net should not be used to update the pseudolabel.

**Comment by degts** [comment]
I have the same doubts about the part of updating pseudolabels.
When I was training my own dataset, the quality of the updated pseudolabels was even lower than the pseudolabels generated by the previously pre-trained network，which resulted in a high dice value on the training set, but a low dice value on the test set instead. If the pseudolabels with low dice value are updated as described in this paper, then the quality of the pesudolabels used for training afterwards will be lower and lower.
I'm not sure I'm right, so if you have a different opinion, please let me know and thank you for your time！
